wiki/Tournaments/GTS/CGTS_2019/en.md

```diff
@@ -61,35 +61,205 @@ The Continental Global Taiko Showdown 2019 is run by various community members.
+
+### North America region
+
+| Seed | Members |
+| :-- | :-- |
+| Top | ![][flag_CA] [janitoreihil](https://osu.ppy.sh/users/3307897), ![][flag_CA] [vladyushko](https://osu.ppy.sh/users/4908773), ![][flag_US] [3san](https://osu.ppy.sh/users/8050850), ![][flag_US] [R J](https://osu.ppy.sh/users/6490509) |
+| High | ![][flag_US] [Whulf](https://osu.ppy.sh/users/11601107), ![][flag_US] [cheese salad](https://osu.ppy.sh/users/6349821), ![][flag_US] [Ney](https://osu.ppy.sh/users/5991961), ![][flag_CA] [kazelone](https://osu.ppy.sh/users/2310786) |
+| Low | ![][flag_US] [closed](https://osu.ppy.sh/users/5116922), ![][flag_CA] [rubies87](https://osu.ppy.sh/users/4949934), ![][flag_US] [Cychloryn](https://osu.ppy.sh/users/6921736), ![][flag_US] [CaptainEChan](https://osu.ppy.sh/users/9536977) |
+| Unseeded | ![][flag_CR] [Hotman](https://osu.ppy.sh/users/7902082), ![][flag_US] [Voltaeyx](https://osu.ppy.sh/users/2715118), ![][flag_US] [driodx](https://osu.ppy.sh/users/9709548), ![][flag_MX] [ZaveryK](https://osu.ppy.sh/users/10913397) |
+
+### Oceania/South East Asia region
+
+| Seed | Members |
+| :-- | :-- |
+| Top | ![][flag_SG] [Blerargh](https://osu.ppy.sh/users/7609510), ![][flag_MY] [\[Zeth\]](https://osu.ppy.sh/users/9912966), ![][flag_PH] [Pochacco](https://osu.ppy.sh/users/2927742), ![][flag_ID] [apaajaboleh10](https://osu.ppy.sh/users/5151647) |
+| High | ![][flag_NZ] [Sparxe](https://osu.ppy.sh/users/5750235), ![][flag_MY] [growATMfarmz](https://osu.ppy.sh/users/12273160), ![][flag_SG] [Element118](https://osu.ppy.sh/users/2283992), ![][flag_AU] [Tsubasa2](https://osu.ppy.sh/users/6835183) |
+| Low | ![][flag_MY] [imnoobatosuosu](https://osu.ppy.sh/users/12086753), ![][flag_AU] [richy2001](https://osu.ppy.sh/users/11499467), ![][flag_AU] [Beat43210](https://osu.ppy.sh/users/5664171), ![][flag_MY] [end_ting](https://osu.ppy.sh/users/8570555) |
+| Unseeded | ![][flag_PH] [Un\_Breaker](https://osu.ppy.sh/users/5670617), ![][flag_MY] [Bedwyr Aorta](https://osu.ppy.sh/users/10875855), ![][flag_AU] [AmateurMonkeyYT](https://osu.ppy.sh/users/8379046), ![][flag_MY] [newme96](https://osu.ppy.sh/users/2782769) |
+
+### South America region
+
+| Seed | Members |
+| :-- | :-- |
+| Top | ![][flag_CL] [Alepat](https://osu.ppy.sh/users/1164931), ![][flag_BR] [HiroK](https://osu.ppy.sh/users/4050738) |
+| High | ![][flag_BR] [Ppus](https://osu.ppy.sh/users/5918857), ![][flag_CL] [Ulqui](https://osu.ppy.sh/users/1263669) |
+| Low | ![][flag_BR] [Coryn](https://osu.ppy.sh/users/2828556), ![][flag_BR] [Foxeru](https://osu.ppy.sh/users/7479684) |
+| Unseeded | ![][flag_AR] [Axer](https://osu.ppy.sh/users/7299864), ![][flag_VE] [JessiChan](https://osu.ppy.sh/users/1368025) |
+
+---
+
+## Match schedule: Round of 32
```

Comment:
with such a massive tournament it's probably not worth adding schedules here until later stages. this tournament looks very well organized so I doubt the players need this (it would be useful for spectators later on)

Reply:
So scrap it until later stages? (e.g. the Global phase)

Reply:
i'd remove it until stages where spectators would be interested in watching livestreams or etc, so that probably means around the finals or semifinals of regional? idk